e-learning-webapps/src/main/webapp/vue-app/main.js

```diff
@@ -1,5 +1,6 @@
 import './initComponents.js';
 
+import * as tutoService from './js/tutoService';
 
```

Comment:
Why did you create a new vue instance ?? ==> export const tutorialsApp = new Vue();